src/components/AdminDashboard/AdminDashboard.js

```diff
@@ -189,129 +146,103 @@ const AdminDashboard = () => {
     setAdding(true);
   };
 
-  const logout = () => {
-    localStorage.removeItem('okta-token-storage', 'okta-cache-storage');
-    push('/');
-    window.location.reload();
```

Comment:
What happened to this logout functionality?

Reply:
![screenshot](https://res.cloudinary.com/hvvmlo7os/image/upload/v1631835201/blue-watch/admindash03_suzpml.png)

It's in the navbar now.